packages/tests/tests/health-manager/utils.ts

```diff
@@ -0,0 +1,21 @@
+import { createDecoder, createEncoder } from "@waku/core";
```

Comment:
this also seems as duplication, can't we re-use from the place it is copied from:
https://github.com/waku-org/js-waku/blob/04bd518210a709b3b01ada5be947e653912201ba/packages/tests/tests/filter/utils.ts#L44

Reply:
To me it seems we can just import const from the file I mentioned and it will be it.

For `nodes.ts` I see that if we just patch original file - it should be fine. 

I agree that we should address https://github.com/waku-org/js-waku/issues/2087 but for current PR let's not introduce too much of the new debt. 

So, I would approach it by:
- moving const to root file or importing them from original;
- importing `node.ts` utils and patching original file with `withoutFilter` property;